docs/DeveloperGuide.md

```diff
@@ -304,16 +304,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `AgentAssist` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: U1. Delete a person**
```

Comment:
Perhaps you can replace `U1.` with `U1 - ...`

Reply:
Thanks for the feedback! Amended in the following commit